VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -1890,5 +1865,6 @@ public void onConfigurationChanged(Configuration newConfig) {
         boolean isLandscape = newConfig.orientation == Configuration.ORIENTATION_LANDSCAPE;
         player.setFullScreen(isLandscape);
         updateController("orientation change");
+        player.requestAccessibilityFocusPausePlay();
```

Comment:
At least on the device I tested on, I saw a regression. When I turned the phone in landscape mode, the accessibility focus disappeared. Again, I can't tell if it's just this device (I can't seem to find the other device I was working for the original PR). But this is to fix that regression....however it was introduced. I get it if we should put in in another PR, but for last night I just put it there.